functions/index.js

```diff
@@ -19,11 +21,19 @@ async function handleRequest(req, res) {
     }
     console.log(req.path);
     res.set('Cache-Control', 'public, max-age=3600, s-maxage=7200');
-    await nuxt.server.app.handle(req, res, (out) => console.log(out));
+    if(dynamicRoutes.includes(req.path)){ 
+      await nuxt.server.app.handle(req, res);
+    }
+    else{
+      console.log('Redirect to 404 page');
+      res.set('X-Cascade', 'PASS')
+      res.status(404).redirect('/404.html')
```

Comment:
I think y'all might need a style enforcer like `prettier` or `standard` I'm seeing `;` sometimes, but not others

Reply:
Probably… we have it in the parent dir… may just need to add it to the functions dir..